week-3/todo-list/index.html

```diff
@@ -3,16 +3,28 @@
   <head>
```

Comment:
It seems the layout and functionality of this todo-list page is not in accordance with the requirements described in the README.md file. Please go through those requirements and make the necessary changes.